src/background/sidebar-injector.js

```diff
@@ -143,6 +145,12 @@ export function SidebarInjector() {
     return url.startsWith('https://bookshelf.vitalsource.com/');
   }
 
+  /** @param {string} url */
+  function isLMSAssignmentURL(url) {
+    const { origin } = new URL(url);
+    return origin.includes('hypothes.is');
```

Comment:
This will match _any_ site hosted on the `hypothes.is` domain or subdomains! I was thinking of something like:

```js
// Matches origins like `lms.hypothes.is`, `qa-lms.hypothes.is`, `lms.ca.hypothes.is`.
origin.match(/\blms\b/) && origin.endsWith('.hypothes.is')
```

Reply:
What do you think to do it on a single regex? `/\blms\b\.*\.hypothes\.is$/.test(origin)`